### PR TITLE
wslay 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by DomT4**

### CODE REVIEW

**Comment by DomT4** [comment]
Afraid it lacks a test right now because me and C have roughly the same relationship as oil and water :crying_cat_face:

**Comment by mikemcquaid** [comment]
@DomT4 Thanks man!

@xu-cheng @bfontaine I've pushed this and if either of you fancy working your test magic on it: feel free!

**Comment by DomT4** [comment]
Thanks Mike!

:heart: to anyone who wishes to add a test later. I'll get around to not hating C one day.

File: Library/Formula/wslay.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  head do
+    url "https://github.com/tatsuhiro-t/wslay.git"
+
+    depends_on "automake" => :build
+    depends_on "autoconf" => :build
+    depends_on "libtool" => :build
+  end
+
+  depends_on :python => :build if MacOS.version <= :snow_leopard && build.with?("docs")
+  depends_on "cunit" => :build
```
Comment (xu-cheng) [inline Library/Formula/wslay.rb]: Is this used in the building? Shouldn't be a `make test` or something to call for the test?

Comment (DomT4) [inline Library/Formula/wslay.rb]: Already fixed locally, I just hadn't pushed the follow up PR yet. Got distracted working on something much bigger and something I'm inevitably likely to end up hating myself for starting :grinning:. Will push the local PR now.

**Pull request closed without merging**